package.json

```diff
@@ -25,6 +25,7 @@
     "moment": "^2.29.1",
     "prettier": "^2.6.2",
     "react": "latest",
+    "react-adobe-fonts": "^0.1.0",
```

Comment:
why do we need this?

Reply:
you're right we don't need it. somehow it got added in here.